src/__tests__/posthog-core.js

```diff
@@ -677,24 +678,121 @@ describe('_loaded()', () => {
         })
     })
 
-    it('captures pageview', () => {
-        given('config', () => ({
-            capture_pageview: true,
-        }))
+    describe('capturing pageviews', () => {
+        beforeEach(() => {
+            given('config', () => ({
+                capture_pageview: false,
+                capture_performance: false, // the default
+            }))
+        })
 
-        given.subject()
+        it('captures not capture pageview if disabled', () => {
+            given.subject()
 
-        expect(given.overrides.capture).toHaveBeenCalledWith('$pageview', {}, { send_instantly: true })
-    })
+            expect(given.overrides.capture).not.toHaveBeenCalled()
+        })
 
-    it('captures not capture pageview if disabled', () => {
-        given('config', () => ({
-            capture_pageview: false,
-        }))
+        describe('when window performance is available', () => {
+            it('captures pageview', () => {
```

Comment:
This test is in the wrong place. This should be one describe up - performance logic does not affect whether we capture a pageview or not.